handlers/handler-loadbalance/src/test/java/io/servicecomb/loadbalance/filter/TestIsolationServerListFilter.java

```diff
@@ -19,24 +19,46 @@
 import java.util.ArrayList;
 import java.util.List;
 
+import org.apache.commons.configuration.AbstractConfiguration;
 import org.junit.After;
 import org.junit.Assert;
 import org.junit.Before;
+import org.junit.BeforeClass;
 import org.junit.Test;
+import org.mockito.Mockito;
 
+import com.netflix.config.DynamicPropertyFactory;
 import com.netflix.loadbalancer.LoadBalancerStats;
 import com.netflix.loadbalancer.Server;
 
+import io.servicecomb.config.ConfigUtil;
+import io.servicecomb.core.Invocation;
+import io.servicecomb.loadbalance.CseServer;
+
 public class TestIsolationServerListFilter {
 
     IsolationServerListFilter IsolationServerListFilter = null;
 
     LoadBalancerStats loadBalancerStats = null;
 
+    @BeforeClass
+    public static void initConfig() throws Exception {
+        ConfigUtil.installDynamicConfig();
```

Comment:
it's better to clean up this config after use, or other tests using archaius may fail